src/client/client.cpp

```diff
@@ -48,6 +48,10 @@ void renderChunks(const std::vector<ChunkDrawable> &chunks,
         }
     }
 }
+
+bool blockIsBreakable(VoxelType voxelType) {
```

Comment:
I'd change this function to something like isBlockSelectable and returning False if the voxelType is any liquid. blockIsBreakable, in my opinion, means if the block is breakable by a player in survival mode (e.g. Bedrock in minecraft... It's selectable but not breakable.)

Reply:
Agreed, have done this now